capstone/src/main/java/com/google/sps/servlets/LoadDataServlet.java

```diff
@@ -22,12 +22,27 @@
 public class LoadDataServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    List<SurveyResponse> surveyResponses = new ArrayList<SurveyResponse>();
-    Query query = new Query("Response");
+    String kind = request.getParameter("kind");
+
```

Comment:
Should we sanity check the value of `kind`earlier and exit out of it the value is not valid?

Reply:
Currently, the value of the kind is hard-coded in the script file